README.md

```
@@ -39,57 +39,37 @@
-| FUSEKI_USER       | admin              | Default fuseki username, change especially if not used locally                                                             | yes              |   |
-| FUSEKI_PASSWORD   | fuseki_password    | Default fuseki password, change especially if not used locally                                                             | yes              |   |
-| POSTGRES_USER     | postgres           | Default postgres username, change especially if not used locally                                                           | yes              |   |
-| POSTGRES_PASSWORD | changeme           | Default postgres password, change especially if not used locally                                                           | yes              |   |
-| POSTGRES_PORT     | 8005               | Port of Postgres DB, only needed for code developers                                                                       | no               |   |
-| FUSEKI_DEV_PORT   | 8003               | Port of Fuseki Triple Store, only needed for code developers                                                               | no               |   |
-| FUSEKI_LIVE_PORT  | 8004               | Port of Fuseki Triple Store (of published vocabularies), only needed for code developers                                   | no               |   |
+| VAR NAME          | Default value      | Description                                                                                                                | Change recommended |   |
+|-------------------|--------------------|----------------------------------------------------------------------------------------------------------------------------|--------------------|---|
+| INSTANCE_NAME     | defaultinstance    | Prefix of docker container names                                                                                           | no                 |   |
+| PROXY_PORT        | 9000               | Port where the proxy is located                                                                                            | no                 |   |
+| PUBLICURL         | http://localhost   | (public) base URL under which the services will be reached                                                                 | no                 |   |
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)